### PR TITLE
[13.0][ddmrp] add method refresh_button in stock.buffer to be called from the buffer views instead of cron_actions

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**

### CODE REVIEW

File: ddmrp/models/stock_buffer.py [hunk 1]
```diff
@@ -1214,6 +1214,11 @@ def cron_ddmrp_adu(self, automatic=False):
         _logger.info("End cron_ddmrp_adu.")
         return True
 
+    def refresh_buffer(self):
+        self.ensure_one()
+        self.cron_actions(only_nfp=False)
```
Comment (LoisRForgeFlow) [inline ddmrp/models/stock_buffer.py:1219]: ensure one is now duplicated in both methods, is is needed in both?

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Why this change? can you elaborate a little bit?

**Review by jgrandguillaume — APPROVED** [review]
LGTM

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**